kolibri/core/assets/src/views/AttemptLogList.vue

```diff
@@ -193,11 +195,24 @@
         return sections.value[currentSectionIndex.value];
       });
 
+      // Computed property for attempt logs of the current section
+      const attemptLogsForCurrentSection = computed(() => {
+        const start = currentSection.value.startQuestionNumber;
+        return currentSection.value.questions.map((_, index) => {
```

Comment:
Could this be simpler?
```
return props.attemptLogs.slice(start, start + currentSection.value.questions.length);
```

I think would give the same effect?

Reply:
I will give it a try!